src/cmd/manage_nsfs.js

```diff
@@ -462,6 +475,12 @@ async function fetch_account_data(argv, from_file) {
         new_name: data.new_name && new SensitiveString(String(data.new_name)),
         new_access_key: data.new_access_key && new SensitiveString(String(data.new_access_key))
     };
+
+    // fs_backend deletion specified with empty string '' (but it is not part of the schema)
+    if (argv.fs_backend === '' && data.nsfs_account_config?.fs_backend !== undefined) {
+        data.nsfs_account_config.fs_backend = undefined;
```

Comment:
same here, line 471 handles this already

Reply:
I will remove it, thanks.